constants.py

```diff
@@ -5,3 +5,7 @@
 for row in DictReader(open("faults.csv")):
     if row["PV Suffix"]:
         CSV_FAULTS.append(row)
+
```

Comment:
I forgot to mention that I wanted to rename this file utils.py lol

Reply:
Done!